packages/styles/scss/patterns/usecases/index.scss

```diff
@@ -3,10 +3,6 @@
 @import './usecases-group';
 @import './usecases-item';
 
-.#{$prefix}--usecases--g100 {
-  @include carbon--theme($carbon--theme--g100) {
-    @include usecases;
-    @include usecases-group;
-    @include usecases-item;
-  }
-}
+@include usecases;
+@include usecases-group;
+@include usecases-item;
```

Comment:
Actually, do we need these if they are already getting included at the bottom of the files themselves?

Reply:
I think this file is exporting all three scss files as one and we still need it. what we inlcuded at the bottom of each scss file is theme specific scss to apply to `themed items`

Reply:
Right, the UseCase story is still importing this index.scss file that includes these 3 files

Reply:
Each file is doing their own `@include` using the `import-once` wrapper like this:

```scss
@include exports('usecases-item') {
  @include usecases-item;
}
```
 
So once you import the file, the mixin already gets fired off at the end of the file, right?

Reply:
But the story is referencing this index file here - don't we need to include all related usecase css files?